def custom_mapper(entry):
    raise Exception
    return entry
